src/components/Charts/StarXYChart.vue

```diff
@@ -8,56 +8,45 @@
 </template>
 
 <script lang="ts">
-import ChartXkcd from "chart.xkcd";
 import { defineComponent, onMounted, onUpdated, ref } from "vue";
+import XYChart, { XYChartData } from "../../packages/xy-chart";
```

Comment:
Let's add a comment here explaining the reason we have to develop our own XYChart

Reply:
This comment doesn't clearly mention *WHY*.  Essentially what's the limitation of the existing xkcd chart, which forces us to rewrite it.